### PR TITLE
Build a react calculator app - Milestone three (Styling)

### PR DESCRIPTION
**Pull request by ampaire**
In this milestone, I worked on styling the UI of the calculator app according to the [requirements](https://github.com/microverseinc/project-react-calculator/blob/master/milestones/MILESTONE_3.md)

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -7,11 +7,32 @@ body {
...
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.app {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto 0 auto;
+  width: 600px;
```
Comment (shloch) [inline src/styles/index.css]: specifications says, you should use `700px` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/700Wide.png)]

File: src/styles/index.css [hunk 2]
```diff
@@ -7,11 +7,32 @@ body {
...
+  flex-direction: row;
+  font-size: 50px;
+  font-weight: bold;
+  height: 30px;
+  padding: 8px;
+}
+
+.btn-group {
+  height: 60px;
+}
```
Comment (shloch) [inline src/styles/index.css]: Requirement talks of using `100px` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png)]

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @ampaire , 

Hello bro...your code is perfectly good, absolutely nothing wrong with it. But for the sake of stcking with requirements, i plead with you to change 2 little values below, and get a thumbs up

Screen Shot 2020-05-20 at 12 24 16 PM
I see your design, i love it..............

### two small changes below please :

File: src/styles/index.css (hunk 2 above)
Comment (ampaire) [reply src/styles/index.css]: Oh, Thank you @shlock, sorry I look like a bro but I am female 😃

**Review by bellom — COMMENTED** [review]
## Hi @ampaire  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by ampaire** [comment]
Hello @bellom, the status of this project in my dashboard is still indicating  **Changes Required**. I don't know how to go about this.

**Comment by bellom** [comment]
Hi @ampaire, you don't have to worry about that, we will look into it.
Can you request for review by clicking the `changes required`?
if yes, do that and submit this `PR` I believe the TSE's would see our conversation and reach out to me.

**Comment by ampaire** [comment]
thanks for reaching out @bellom, I tried submitting again for review but I get this error saying I have no access to the page.

**Comment by bellom** [comment]
Ok, kindly share your full name.
I will forward it to the dev guys.

**Comment by ampaire** [comment]
Phemia (Effie) Ampaire

**Comment by bellom** [comment]
**SENT**!
kindly wait for them to work on it, continue with your project.
Best.

File: src/styles/index.css (hunk 2 above)
Comment (shloch) [reply src/styles/index.css]: ooooh sorry about that, you look like a female oooh....i did not look well enough...my bad!
I guess i was misled by the name instead, not the face

**Review by shloch — COMMENTED** [review]
On this basis of the discussion, i guess i just have to validate lol
accomplished

**Pull request merged**